hammerdb/build-and-run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Please install `shellcheck` and fix/explicitely ignore the warnings it gives. Bash is nice, but very brittle if you don't follow shellcheck.